tools/deb/istio.mk

```diff
@@ -0,0 +1,83 @@
+.PHONY: docker.istio.deb istio.deb sidecar.deb ${OUT}/istio-sidecar.deb
```

Comment:
I'm not sure if it's appropriate for "${OUT}/istio-sidecar.deb" to be in this list.  Normally .PHONY is used for targets that won't (or normally shouldn't) exist, and this file will exist.  However, .PHONY can also be used to always force a rebuild.  If that's the intent the I suppose .PHONY is alright though ideally enough dependencies are captured now that we don't need to force rebuilds.

Reply:
I added it because I was not sure if we have all deps - but you are right, I think we do capture all deps now.